test/e2e/arktos/multi_tenancy/testdata/regular_crd_resource.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: fantasy.wonderland.com/v1
+kind: NeverSharingSweetDream
+metadata:
+  name: wild-wild-west
```

Comment:
I enjoyed these

Reply:
Hahaha. Let's work together to spread some sunshine to the dull life of a code worker.